apps/common-app/src/examples/RuntimeTests/tests/animations/withDelay/addDelays.test.tsx

```diff
@@ -158,8 +161,10 @@ describe('Add three delays', () => {
       );
 
       expect([{ width: 0 }, { width: 0 }, ...updatesOneDelay]).toMatchSnapshots(updates);
-      expect(updatesOneDelay).toMatchNativeSnapshots(nativeUpdatesOneDelay);
-      expect(updates).toMatchNativeSnapshots(nativeUpdates);
+      if (componentType !== 'INLINE' || Platform.OS !== 'android') {
```

Comment:
There are quite a few workarounds in this PR. I understand that they fix some issues, but let me take some time to consider a potential solution to fix the root cause of the problem.